environments/openshift/obs.jsonnet

```diff
@@ -340,6 +344,38 @@ local up = (import '../../components/up.libsonnet');
       },
     },
 
+    storeCache+: {
+      local scConfig = self,
+      image:: '${THANOS_STORE_MEMCACHED_IMAGE}:${THANOS_STORE_MEMCACHED_TAG}',
+      exporterImage:: '${THANOS_STORE_MEMCACHED_IMAGE}:${THANOS_STORE_MEMCACHED_TAG}',
+      version: '${THANOS_STORE_MEMCACHED_TAG}',
+      exporterVersion: '${THANOS_STORE_MEMCACHED_EXPORTER_TAG}',
+      replicas: '${{THANOS_STORE_MEMCACHED_REPLICAS}}',
```

Comment:
is this variable set?

Reply:
What do you mean? `{{}}` for casting values to integers, if you're asking that.

Reply:
wow did not notice that.

I mean that you define all those variables below but not this one

Reply:
Nice catch, let me add that.